app/modules/encounters/models.py

```diff
@@ -147,15 +147,23 @@ def delete_from_edm(self, current_app):
     # given edm_json (verbose json from edm) will populate with houston-specific data from feather object
     # note: this modifies the passed in edm_json, so not sure how legit that is?
     def augment_edm_json(self, edm_json):
+        edm_json['createdHouston'] = self.created.isoformat()
+        edm_json['updatedHouston'] = self.updated.isoformat()
+        from app.modules.users.schemas import PublicUserSchema
+
+        usch = PublicUserSchema(many=False)
```

Comment:
I would really prefer user_scheme as a variable name, usch sounds like a sneeze

Reply:
and yet you didnt care about `sch` a few lines below?  :smirk:   i can change _both_.

Reply:
cared yes, spotted no :)